packages/transformers/js/core/src/env_replacer.rs

```diff
@@ -41,6 +42,25 @@ impl<'a> Fold for EnvReplacer<'a> {
       }
     }
 
+    if let Expr::Bin(ref binary) = node {
```

Comment:
We should probably check that the binary expression is actually an `in`. There are other binary expression operators that shouldn't be replaced.